src/constants/properties.constant.ts

```diff
@@ -418,4 +418,64 @@ export const PROPERTIES = {
       },
     },
   },
+  REPAYMENT: {
+    DEFAULT: {
+      primaryScheduleIndicator: 'true',
+      involvedParty: {
+        partyIdentifier: '00000000',
+      },
+      lenderType: {
+        lenderTypeCode: '100',
+      },
+      accountSequence: '1',
+      leadDays: 5,
+      nextDueBusinessDayAdjustmentType: {
+        loanSystemBusinessDayAdjustmentTypeCode: 'M',
+      },
+      nextAccrueBusinessDayAdjustmentType: {
+        loanSystemBusinessDayAdjustmentTypeCode: 'M',
+      },
+      billingPeriod: 0,
+      collectionInstructionMethod: {
+        collectionInstructionMethodCode: '',
+      },
+      billFormatType: {
+        billFormatTypeCode: '',
+      },
+      mailingInstructionType: {
+        mailingInstructionTypeCode: '',
+      },
+      spreadToInvestorsIndicator: true,
+      balloonPaymentAmount: 0,
+      loanPrePaymentType: {
+        loanPrePaymentTypeCode: 2,
+      },
+    },
+    INT: {
+      billingScheduleType: {
+        billingScheduleTypeCode: 'A',
+      },
+      billingSequenceNumber: 2,
+    },
+    PAC: {
+      primaryScheduleIndicator: 'false',
```

Comment:
@francescastocco has looked into this and found that `'false'` is interpreted as `false` and `'true'` as `true` by ACBS fortunately, so there are no issues here. However, sending booleans to ACBS also works correctly as is the documented approach so I've asked @francescastocco to change `PrimaryScheduleIndicator` to use booleans instead of strings